t.in.eoarchive.py

```diff
@@ -0,0 +1,621 @@
+    cmd = grass.Popen(cmd_str, shell=True, stdout=subprocess.PIPE,
+                      stderr=subprocess.PIPE)
+    resp = cmd.communicate()
+    resp_text = ""
+    for resp_line in resp:
+        resp_text += resp_line.decode("utf-8")
+    empty_strings = [f"<{name}> is empty",
+                     "Input raster does not overlap current computational region"]
+    if any(string in resp_text for string in empty_strings):
+        map_empty = True
+        grass.warning(_("Only no-data values found in current region for input "
+                        f"raster <{input}>"))
+    else:
+        map_empty = False
+    grass.run_command(
+        "r.semantic.label",
+        map=name,
+        semantic_label=semantic_label,
+        operation="add")
+    import_dict = {"name": name, "datetime": scene_datetime,
+                   "map_empty": map_empty, "band": semantic_label}
+    return import_dict
+
+
+def import_parallel(import_parallel_list, num_processes):
+    """ Function that sets up and runs a pool for parallel processing of
+        importing. Calls the import_raster function.
+    Args:
+        import_parallel_list(list): List with dictionaries as items that match
+                                    the required input pattern of the
+                                    import_raster function
+        num_processes(int): Number of parallel processes to use
+    Returns:
+        output(list): List of dictionaries corresponding to the output format
+                      of the import_raster function
+    """
+    pool = Pool(processes=num_processes)
+    output = pool.map(import_raster, import_parallel_list)
+    return output
+
+
+def main():
+
+    global TMPLOC, SRCGISRC, TGTGISRC, GISDBASE
+    global rm_rasters, rm_vectors
+    global collection_params
+
+    # parameters
+    start = options['start']
+    end = options['end']
+    if not start:
+        start = "2015-07-01"
+    if not end:
+        end = "today"
+    mountpoint = options["mountpoint"]
+    collection = options["collection"]
+    bands = options["bands"].split(",")
+    output = options["output"]
+    archive = options["archive"]
+
+    start_date, end_date = check_start_end(start, end)
+    grass.message(_("Filtering available scenes..."))
+    if archive == "eolab":
+        scenes_to_import = browse_eolab_collection(
+            collection=collection,
+            bands=bands,
+            mountpoint=mountpoint,
+            start_date=start_date,
+            end_date=end_date)
+    if len(scenes_to_import) == 0:
+        grass.fatal(_("No scenes matching the spatial and temporal filter "
+                      "found. Exiting..."))
+
+    # Importing using the Pool method
+    nprocs, used_ram = test_nprocs_memory()
+    ram_per_proc = int(used_ram / nprocs)
+    # if memory is >= 100000 GDAL interprets it as bytes, not MB.
+    if ram_per_proc >= 100000:
+        ram_per_proc = ram_per_proc * 1000000
```

Comment:
```suggestion
        ram_per_proc = ram_per_proc * 100000
```
too many zeroes to count...

Reply:
This has now been replaced by `1048576` (1024 * 1024)